testing/tests/configs_tree_view.py

```diff
@@ -9,228 +9,200 @@
 Tests specifically focused on the various tree view configurations.
 """
 from __future__ import unicode_literals
+
 import os
+import platform
 import re
+from pathlib import Path
 from textwrap import dedent
 
 from testing.base import ExhaleTestCase
 from testing.decorators import confoverrides
 
+this_file_dir = Path(__file__).parent.absolute()
+"""The path to the directory containing this file."""
+
+class_data_root = this_file_dir / "configs_tree_view_data" / "classes"
+"""The class view hierarchy test validation data root for tree view."""
+
+file_data_root = this_file_dir / "configs_tree_view_data" / "files"
+"""The file view hierarchy test validation root for tree view."""
+
+tree_view_keys = {"default_rst_list", "collapsible_lists", "bootstrap"}
+"""Available kinds of tree views to validate against."""
+
+
+def read_default_data(root: Path, file_name: str, is_html: bool) -> str:
```

Comment:
TODO: hide all of this away in `configs_tree_view_data/__init__.py` (to be created)

Reply:
done